api-reference/10 UI Widgets/dxFileUploader/1 Configuration/dialogTrigger.md

```diff
@@ -5,7 +5,7 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the HTML element which invokes the file upload dialog.
+Specifies the HTML element a click on which invokes the file upload dialog.
```

Comment:
This does not make sense. This word order makes the sentence awkward. 

Why did you change the original?

Reply:
You click a HTML element and this invokes the dialog. This option specifies this element.

Reply:
Yes, but surely "the HTML element which invokes the file upload dialog" already implies this. Your version sounds awkward, especially the part about "a click on which"

Reply:
ok